core/load_copyright_map.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import sys
+import datetime
+from core.models import Copyright
+from core.models import LccnDateCopyright
+
+def check_uri(_uri):
+    try:
+        result = Copyright.objects.filter(uri=_uri)
+        return result.exists()
+    except Exception as e:
+        _logger.exception(e)
+
+def loadCopyrightMap(inputfilename):
+
+    _logger = logging.getLogger(__name__)
+    try:
+        f = open(inputfilename, 'r')
+        for line in f:
+            arr = line.split("\t")
+            if len(arr) == 4:
```

Comment:
Could we raise an exception if this fails?  And probably wrap the thing in a transaction so we can rollback since there's no way to make changes at the moment.